csrf/api/v1/views.py

```diff
@@ -27,8 +28,11 @@ class CsrfTokenView(APIView):
     """
 
     authentication_classes = (JwtAuthentication,)
-    permission_classes = (IsAuthenticated,)
```

Comment:
If not requiring `IsAuthenticated`, why is `JwtAuthentication` still listed as an authentication class?

Reply:
I will test if this affects the get_token() call and remove or comment accordingly.

Reply:
Seems to work without JwtAuthentication, so I removed this.